rclpy/rclpy/__init__.py

```diff
@@ -59,7 +59,12 @@
     from rclpy.node import Node  # noqa: F401
 
 
-def init(*, args: Optional[List[str]] = None, context: Context = None) -> None:
+def init(
+    *,
+    args: Optional[List[str]] = None,
+    context: Context = None,
+    domain_id: Optional[int] = None
```

Comment:
domain_id should be checked if it is greater than or equal to zero?

Reply:
Yes. thanks.  
I supplement checking codes.